src/nvim/testdir/test_fileformat.vim

```diff
@@ -32,6 +31,251 @@ func Test_fileformat_autocommand()
   bw!
 endfunc
 
+" Convert the contents of a file into a literal string
+func s:file2str(fname)
+  let b = readfile(a:fname, 'B')
+  let s = ''
+  for c in b
+    let s .= nr2char(c)
+  endfor
+  return s
+endfunc
+
+" Concatenate the contents of files 'f1' and 'f2' and create 'destfile'
+func s:concat_files(f1, f2, destfile)
+  let b1 = readfile(a:f1, 'B')
+  let b2 = readfile(a:f2, 'B')
+  let b3 = b1 + b2
+  call writefile(b3, a:destfile, 'B')
```

Comment:
```suggestion
  call writefile(b3, a:destfile)
```

looks like a mistake missed due to vim's lack of flag validation here?

imo better to change the test instead

Reply:
Indeed this flag is nowhere documented.

Reply:
Sweet, thanks @seandewar , I've done that.